devices/YarpRobotLoggerDevice/app/robots/ergoCubSN001/yarp-robot-logger.xml

```diff
@@ -19,7 +19,7 @@ BSD-3-Clause license. -->
   <param name="port_prefix">/yarp-robot-logger</param>
   <param name="text_logging_subnames">("ergoCubSN001/yarprobotinterface")</param>
   <param name="code_status_cmd_prefixes">("ssh ergocub@10.0.2.2" "ssh ergocub@10.0.2.3")</param>
-  <param name="maximum_admissible_time_step">1.0</param>
+  <param name="maximum_admissible_time_step">60.0</param>
```

Comment:
I would completely remove this parameter so we always get the data

Reply:
Done in https://github.com/ami-iit/bipedal-locomotion-framework/pull/869/commits/11a95f79a307503389e11074145218b07ca1774c